.gitignore

```diff
@@ -1,9 +1,12 @@
 # FILES
 **/**/.pyc
 .DS_Store
+passwords.txt
 
```

Comment:
Kindly ignore `db.sqlite3` by including it to ignore files - it's auto generated and as such not needed in VC.